bastionzero/environment/resource_environment_test.go

```diff
@@ -93,6 +93,60 @@ func TestAccEnvironment_Disappears(t *testing.T) {
 	})
 }
 
+func TestAccEnvironment_Name(t *testing.T) {
+	ctx := context.Background()
+	rDesc := acctest.RandomName()
+	name1 := "development_environment"
+	name2 := "development"
+	resourceName := "bastionzero_environment.test"
+	var env environments.Environment
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(ctx, t) },
+		ProtoV6ProviderFactories: acctest.TestProtoV6ProviderFactories,
+		CheckDestroy:             testAccCheckEnvironmentDestroy,
+		Steps: []resource.TestStep{
+			// Verify create works for a config that sets description
+			{
+				Config: testAccEnvironmentConfigName(name1),
+				Check: resource.ComposeTestCheckFunc(
+					// Check environment exists at BastionZero
+					testAccCheckEnvironmentExists(resourceName, &env),
+					// Check environment stored at BastionZero looks correct
+					testAccCheckEnvironmentAttributes(&env, &expectedEnvironment{
+						Name:        &name1,
+						Description: &rDesc,
```

Comment:
Remove checking description since this test focuses only on name

Reply:
[ec55904](https://github.com/bastionzero/terraform-provider-bastionzero/pull/35/commits/ec559047f3e63457fc457c3e04329f2b68a23d0e)